<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mass Density | Neutron Density | Mass</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QRadioButton" name="mass_density_radio_button">
        <property name="text">
         <string>Mass Density</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="mass_density_line_edit"/>
         </item>
         <item>
          <widget class="QLabel" name="mass_density_label">
           <property name="text">
            <string>g/cc</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>112</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="mass_density_error_message">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Missing Chemical Formula to Update Mass Density!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>441</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QRadioButton" name="number_density_radio_button">
        <property name="text">
         <string>Number Density</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="number_density_line_edit"/>
         </item>
         <item>
          <widget class="QLabel" name="number_density_units">
           <property name="text">
            <string>atom/A</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>69</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="number_density_error_message">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Missing Chemical Formula to Update Number Density!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>441</width>
        <height>26</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QRadioButton" name="mass_geometry_radio_button">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Mass</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mass_line_edit"/>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>g</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>geometry:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="geometry_label">
               <property name="text">
                <string>Cylinder</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Volume: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="volume_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>N/A</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="volume_units">
               <property name="text">
                <string>cm3</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="mass_error_message">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Geometry Dimensions Have Not Been Defined! </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>441</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ok">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>mass_density_radio_button</tabstop>
  <tabstop>number_density_radio_button</tabstop>
  <tabstop>mass_geometry_radio_button</tabstop>
  <tabstop>mass_density_line_edit</tabstop>
  <tabstop>number_density_line_edit</tabstop>
  <tabstop>mass_line_edit</tabstop>
  <tabstop>ok</tabstop>
  <tabstop>cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>number_density_radio_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radio_button_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>number_density_line_edit</sender>
   <signal>textEdited()</signal>
   <receiver>MainWindow</receiver>
   <slot>number_density_value_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mass_line_edit</sender>
   <signal>textEdited()</signal>
   <receiver>MainWindow</receiver>
   <slot>mass_value_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mass_geometry_radio_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radio_button_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mass_density_radio_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radio_button_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mass_density_line_edit</sender>
   <signal>textEdited()</signal>
   <receiver>MainWindow</receiver>
   <slot>mass_density_value_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>accept()</slot>
  <slot>radio_button_changed()</slot>
  <slot>number_density_value_changed()</slot>
  <slot>mass_value_changed()</slot>
  <slot>mass_density_value_changed()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
